bpaf_core/src/visitor/explain_unparsed.rs

```diff
@@ -127,6 +130,25 @@ impl<'a> ExplainUnparsed<'a> {
         }
         None
     }
+    fn is_redundant(&self, parsed: &[Name], unparsed: Name) -> Option<Error> {
+        let unparsed_info = self.all_names.get(&unparsed)?;
+        if unparsed_info.in_many {
+            return None;
+        }
+        for p in parsed {
```

Comment:
But I think the rest can simply be `parsed.contains(&unparsed)` of some sort - "if name can be parsed only once (first condition) and we already parsed it (this condition)" then we tell the user not to pass it twice...

Reply:
Do we have the information regarding only once stored right now?

Reply:
`!info.in_many` would be it I guess. Let me show an example.

```rust
#[derive(Bpaf, Clone, Debug)]
#[bpaf(options)]
struct Options {
    a: usize,
    b: usize,
    #[bpaf(argument, many)]
    c: Vec<usize>,
}
```

So say we have this parser. It takes arguments `-a=0` and `-b=0` once, `-c=0` multiple times.

If user passes `-a0 -b3` -c1 -c2 -c3 -c4 -a1` parser will run up to `-c4` and will give up on `-a1`.

Visitor gets `-a0 -b3` -c1 -c2 -c3 -c4` in parsed, `-a1` in unparsed (well, just names I guess)... 

To explain that name `-a` can be used only once we only need to look at name `-a` that it is not inside of `many` context - it is not.